README.md

```diff
@@ -25,14 +40,31 @@ Features available:
 
 Features to be added: check the [Issues](https://github.com/windup/windup-api/issues) for enhancements already planned and feel free to add your request for new features if it's not there yet.  
 
+# Technical Considerations
+
 ## REST endpoints
 
 All the available endpoints are described with OpenAPI specifications in the [openapi.yaml](src/main/resources/META-INF/openapi.yaml) file.  
 It can be analyzed, for example, with the online [Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/windup/windup-api/main/src/main/resources/META-INF/openapi.yaml)  
-A Swagger UI is also embedded in the Windup API and accessible from the provided sample page as described below in the [Try with Swagger UI](#try-with-swagger-ui) paragraph.  
+A Swagger UI is also embedded in the Windup API and accessible from the provided sample page as described below in the [API with Swagger](#api-with-swagger) paragraph.  
+
+## Configuration parameters
+
+### Max size for uploaded applications
+
+Windup API default configuration allows the upload of applications up to 100 MB.  
+This size can be changed applying a different value to the `QUARKUS_HTTP_LIMITS_MAX_BODY_SIZE` environment variable executing:
+```shell
+kubectl set -n windup env deployment windup-api QUARKUS_HTTP_LIMITS_MAX_BODY_SIZE="<new_value>"
```

Comment:
My only comment is what format the new value should be in. For example if the user wanted to increase the limit to 200 MB would the value be "200 MB".